docs/plugins/composable_resources.md

```diff
@@ -197,6 +197,13 @@ To modify an existing passthru resource to be written to only after other eligib
 irods@hostname:~/ $ iadmin modresc newResc context 'write=0.4;read=1.0'
 ```
 
+Setting a `read` or `write` weight to `0` will prevent that action from occurring on the branch of the resource tree under the passthru (because a `0` vote signals that it cannot service the incoming request).  The following will 'turn off' the resource `newResc` (a policy-based way of marking a resource "down"):
+
+
```

Comment:
These lines can be removed.

Reply:
removed one - kept one empty line before codeblock.